Reemit.Common.UnitTests/SharedReaderTests.cs

```diff
@@ -82,66 +82,100 @@ public void ReadBytes_Called_EmptyBufferKeepsRelativeOffsetAndOffset ()
         using var binaryReader = new BinaryReader(stream);
         using var sharedReader = new SharedReader(sharedReaderOffset, binaryReader, new object());
         var actualUnmanagedValues = new T[expectedUnmanagedValues.Length];
+        var isRangeMapped = false;
+        var expectedPositions = new int[expectedUnmanagedValues.Length];
+        var actualRangeMaps = new IRangeMapped[expectedUnmanagedValues.Length];
 
         // Act
         for (var i = 0; i < actualUnmanagedValues.Length; i++)
         {
-            actualUnmanagedValues[i] = readUnmanaged(sharedReader);
+            var expectedPosition = sharedReader.Offset;
+            var actualValue = ((Delegate)readUnmanaged).DynamicInvoke(sharedReader)!;
```

Comment:
Nitpicking. The `(Delegate)` cast seems unnecessary.

Reply:
Definitely, this was a holdover from my refactoring to `Delegate`. Fixed.